### PR TITLE
improved null check

### PR DESCRIPTION
**Pull request by paddeo**

### CODE REVIEW

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Comment by paddeo** [comment]
retest this please

**Comment by entando-service** [comment]
Something went wrong. Investigate!

File: plugins/entando-plugin-jpkiebpm/src/main/java/com/agiletec/aps/system/services/page/KiePageManager.java [hunk 1]
```diff
@@ -48,9 +48,9 @@ public void removeWidget(String pageCode, int pos) throws ApsSystemException {
         try {
             Widget widget = this.getDraftPage(pageCode).getWidgets()[pos];
             super.removeWidget(pageCode, pos);
-            String stringId = widget.getConfig().getProperty(KieBpmSystemConstants.WIDGET_PARAM_DATA_UX_ID);
+            String stringId = widget != null && widget.getConfig() != null ? widget.getConfig().getProperty(KieBpmSystemConstants.WIDGET_PARAM_DATA_UX_ID) : null;
```
Comment (joewhite101) [inline plugins/entando-plugin-jpkiebpm/src/main/java/com/agiletec/aps/system/services/page/KiePageManager.java]: See prior comment on the same(?) PR. Change this to an if statement with a setter. Tough to read multi line trinary operators.

**Comment by joewhite101** [comment]
Retest this please

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Comment by pietrangelo** [comment]
retest this please

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Comment by pietrangelo** [comment]
retest this please

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Comment by pietrangelo** [comment]
retest this please

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Comment by pietrangelo** [comment]
retest this please

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Comment by eugeniosant** [comment]
retest this please

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Comment by pietrangelo** [comment]
retest this please

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Comment by pietrangelo** [comment]
retest this please

**Comment by entando-service** [comment]
All is Well

**Pull request merged**